crates/bevy_reflect/bevy_reflect_derive/src/container_attributes.rs

```diff
@@ -220,19 +220,22 @@ impl ReflectTraits {
     ///
     /// If `Hash` was not registered, returns `None`.
     pub fn get_hash_impl(&self, bevy_reflect_path: &Path) -> Option<proc_macro2::TokenStream> {
+        let option = quote!(::core::option::Option);
+        let any = quote!(::core::any::Any);
```

Comment:
You could potentially also do something like:

```rust
pub(crate) struct FQOption;

impl ToTokens for Option {
  fn to_tokens(&self, tokens: &mut TokenStream) {
    quote!(::core::option::Option).to_tokens(tokens);
  }
}

// ...

quote!(let foo: #FQOption<i32> = None;);
```

Which would avoid having to add these lines everywhere we need fully-qualified traits.

Thoughts on this? Would it be worth doing? Or is it not that big a deal to just include these lines where needed?

Reply:
This method is much better. It should reduce repitition across many files. I am working on this.